src/Bicep.Core/TypeSystem/Providers/Extensibility/ExtensionResourceTypeFactory.cs

```diff
@@ -241,11 +248,17 @@ private static ResourceFlags ToResourceFlags(Azure.Bicep.Types.Concrete.Resource
 
         private static ResourceScope ToResourceScope(Azure.Bicep.Types.Concrete.ScopeType input)
         {
-            if (input == Azure.Bicep.Types.Concrete.ScopeType.Unknown)
+            if (input == Azure.Bicep.Types.Concrete.ScopeType.None)
```

Comment:
This semantically might not make sense, but the behavior is the same and is backwards compatible.

By renaming `ScopeType.Unknown` to `ScopeType.None` the flag has all bits set to 0. Since it's an Enum, the flag names don't affect functionality as long as their numeric values stay the same.